kubernetes/ama-logs.yaml

```diff
@@ -384,7 +384,7 @@ spec:
         #         - NET_ADMIN
         #         - NET_RAW
```

Comment:
update from azure-mdsd-1.17.0 to azure-mdsd-1.23.3 in all places in the ama-logs.yaml file

Reply:
updated